tests/forker.c

```diff
@@ -24,7 +24,9 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.  */
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
+#if defined(__linux__)
 #include <alloca.h>
+#endif
```

Comment:
According to manpage, `alloca.h` is included by stdlib.h, so we do not need this on Linux either. I tried setting up FreeBSD CI and tried on both FreeBSD and Linux, it build without this header:

Linux: https://travis-ci.com/am11/libunwind/builds/133997124
FreeBSD: https://cirrus-ci.com/task/6552145913970688
Commit: https://github.com/am11/libunwind/commit/5bc1d67f7c2fcf4225f490fbcd68b27279840624

maybe we can just remove it? :)